### ISSUE TITLE
IB Designables Error on Swift 2.3

### ISSUE THREAD

**Issue by monshipouri** [prolificinteractive/Caishen#92]
Hi,

When I add CardTextField class to my UITextField, I get IB Designable errors.

![errors](https://cloud.githubusercontent.com/assets/1050041/19623735/f1213ad8-988b-11e6-9577-d620fbdf5a83.png)

**Comment by DannyVancura** [comment]
Hi, thanks for letting us know about this issue!
From what I can find online, this seems to be an issue with Xcode 8 and IBDesignable in Swift 2.3 in general: https://forums.developer.apple.com/thread/63237
Will look more into this in the coming week.

**Comment by monshipouri** [comment]
The way I fixed it was to create another class and Inherit from CardTextField, Like:

```
import Foundation
import Caishen
class SMCardTextField: CardTextField {
 ...
}
```

I hope it can help. Thanks

**Comment by DannyVancura** [comment]
Closed with #102

**Comment by monshipouri** [comment]
Thanks
